package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apostrophe-pieces-export",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
This seems like a minor version change.

Reply:
@abea is correct, new features must always bump the minor (middle) version number.